html/views/common.js

```diff
@@ -20,6 +20,7 @@ function jammer(k, v) {
 	};
 	$(".Team" + id + " .Lead").toggleClass("HasLead", (leadJammer && !starPass));
 	$(".Team" + id).toggleClass("HasJammerName", (jn != ""));
+	$(".Team" + id).toggleClass("IsStarPass", (starPass != false));
```

Comment:
Why not just `starPass` ?

Reply:
True, that was outdated as it resultetd from a previous version, wich I had.